test/data/test-ini

```diff
@@ -21,7 +21,9 @@
 [string]
 	simple = A simple string
 	quoted = "A quoted string"
-	quoted_with_quotes = "A string "with quotes""
+	quoted_with_quotes = "A string \"with quotes\""
+	quoted_with_comment = "A quoted string" # a comment
+	escaped_quotes = String \"with quotes\"
```

Comment:
Tests are lacking a bit, can you expand them to cover the corner cases?
A few examples I can think of right now:
* A non-quoted string with a comment
* String with a comment that starts with a semicolon
* A quoted string that contains comment characters (`;#`)
* String that ends with a backslash
* Unknown escape sequences
* A non-quoted string that has quotes in the middle (e.g. `A "string#"` should yield `A string#`)